pkg/kn/commands/source/ping/ping.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	MaxUpdateRetries = 3
+	MaxUpdateRetries = 5
```

Comment:
@vyasgun one more leftover const, otherwise it's OK.

Reply:
Thanks for pointing out. Fixed the remaining consts